src/main/kotlin/de/bund/digitalservice/useid/config/WellKnownProperties.kt

```diff
@@ -0,0 +1,51 @@
+package de.bund.digitalservice.useid.config
+
+import org.springframework.boot.context.properties.ConfigurationProperties
+import org.springframework.stereotype.Component
+import org.springframework.validation.annotation.Validated
+import javax.validation.Valid
+import javax.validation.constraints.NotEmpty
+
+@Component
+@ConfigurationProperties(prefix = "wellknown")
+@Validated
+class WellKnownProperties {
```

Comment:
I like the new package structure. Why not move this to the `wellknown` package? :) It's not a global config but only needed in the package.